docker/legacy-auth/env.template

```diff
@@ -15,11 +15,23 @@ AUTH0_SECRET=
 AUTH0_BASE_URL=https://example.fiftyone.ai
 
 # This should be set to the URI your end-users will use to connect to the API
-#  This could be the same as AUTH0_BASE_URL
+#  This could be the same as AUTH0_BASE_URL if you are using path-based routing
 FIFTYONE_API_URI=https://example-api.fiftyone.ai
 
 # This should be a MongoDB Connection String for your database
 FIFTYONE_DATABASE_URI="mongodb://username:password@mongodb-example.fiftyone.ai:27017/?authSource=admin"
+# If you are using a different MongoDB Connection String for your CAS database,
+#  set it here
+# CAS_MONGODB_URI="mongodb://username:password@mongodb-cas-example.fiftyone.ai:27017/?authSource=admin"
+
+# FIFTYONE_AUTH_SECRET is a random string used to authenticate to the CAS service
+# This can be any string you care to use generated by any mechanism you prefer.
+# You could use something like:
+#  `cat /dev/urandom | LC_CTYPE=C tr -cd '[:graph:]' | head -c 32`
+#  to generate this string.
```

Comment:
it's probably worth noting that we suggest a different random string creation mechanism about 18 lines up.

This is just the one I found when I googled "bash generate random string"